site/helm-integration.md

```diff
@@ -7,6 +7,7 @@ menu_order: 90
   * [The `HelmRelease` custom resource](#the-helmrelease-custom-resource)
     + [Using a chart from a Git repo instead of a Helm repo](#using-a-chart-from-a-git-repo-instead-of-a-helm-repo)
       - [Notifying Helm Operator about Git changes](#notifying-helm-operator-about-git-changes)
+      - [Reinstalling a Helm release](#reinstalling-a-helm-release)
```

Comment:
I think this should be one level higher (as it is not related to charts from Git repositories).

Reply:
Hmm the `using-a-chart-from-a-git-repo-instead-of-a-helm-repo` doesn't even exists